README.md

```diff
@@ -280,11 +258,9 @@ I would like to thank...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
-
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Really good that you added the `correct` **MIT** license file but your `LICENSE` file does contain the placeholder which you were supposed to change with your own personal information to make it more legit, because including the **MIT** *License* in your `GitHub` repository **README** is crucial for several reasons. Firstly, the **MIT** License is a permissive `open-source` license that grants users the freedom to use, modify, and distribute the software without legal restrictions. By choosing the *MIT* License 🗒️, you are encouraging collaboration, inviting other developers to contribute, and fostering a vibrant open-source community. Additionally, using the *MIT* License helps protect your work by clearly defining the terms and conditions under which it can be used, providing a level of legal 🚨 clarity and ensuring that proper attribution is given. Furthermore, the widespread popularity and familiarity of the *MIT* License make it easy for others to understand and comply with the licensing requirements, which ultimately promotes the adoption and usage of your software 💻 .

![image](https://github.com/Obote/Leaderboard/assets/66476329/778a5603-9eb9-4962-ae79-e9a37697db9d)